platform/golden/cfg/preseed-debian-11.cfg

```diff
@@ -50,17 +51,36 @@ tasksel tasksel/skip-tasks  string standard
 tasksel tasksel/first multiselect openssh-server
 popularity-contest popularity-contest/participate boolean false
 
-d-i pkgsel/include string curl wget openssh-server neovim sudo cloud-init
+d-i pkgsel/include string curl wget openssh-server neovim sudo cloud-init ovmf grub-efi-amd64
 d-i pkgsel/language-pack-patterns   string
 d-i pkgsel/install-language-support boolean false
 d-i pkgsel/upgrade select full-upgrade
 
+# -------- Hardware Firmware --------
+d-i hw-detect/load_firmware boolean true
+
 ###########
 ## Use XFS
 
-d-i partman/default_filesystem string xfs
+# Force UEFI booting ('BIOS compatibility' will be lost). Default: false.
+d-i partman-efi/non_efi_system boolean true
+
+# Ensure the partition table is GPT - this is required for EFI
+d-i partman-partitioning/choose_label select gpt
+d-i partman-partitioning/default_label string gpt
+#d-i partman/default_filesystem string xfs
+
+# Use raw partitions 
 d-i partman-auto/method string regular
 
+# # Use LVM
```

Comment:
I don't want to use LVM. It vastly vastly complicates the passing of memory. I wasn't very clear in the requirements.

LVM introduces a ton of complexity into the software. There are many technical reasons for the `xfs` choice.

Can you revert the lvm part of the work?
Software runs on a filesystem. The filesystem runs on a block device. Software lodats

Reply:
the LVM code was left there for the moment because I had run into a few references that "implied" that LVM was needed for UEFI.  I don't believe that's necessary, I'll remove these pieces.